Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -341,12 +371,14 @@ class Client(BaseClient):
     Client to use in the ServiceNow integration. Overrides BaseClient.
     """
 
-    def __init__(self, server_url: str, username: str, password: str, verify: bool, proxy: bool, fetch_time: str,
-                 sysparm_query: str, sysparm_limit: int, timestamp_field: str, ticket_type: str, get_attachments: bool):
+    def __init__(self, server_url: str, sc_server_url: str, username: str, password: str, verify: bool, proxy: bool,
```

Comment:
We don't want another client for the service catalog?

Reply:
Since it is one more arg in he ini I think we can do without. Do you think it is needed?

Reply:
might be overkill. Just I'm not sure that extra arg is very clear. We should add some comments then.

Reply:
Added in `send_request` and in the relevant client methods.